core/controllers/base.py

```diff
@@ -411,6 +411,16 @@ def vaidate_and_normalize_args(self):
             arg: normalized_value.get(arg) for arg in request_arg_keys
         }
 
+        default_value_keys = list(
+            set(normalized_value.keys()) -
+            set(payload_arg_keys + request_arg_keys)
+        )
+        for arg in default_value_keys:
+            if request_method in ['GET', 'DELETE']:
+                self.normalized_request[arg] = normalized_value.get(arg)
+            else:
+                self.normalized_payload[arg] = normalized_value.get(arg)
```

Comment:
Even in the POST method, there is a chance that the value can be a part of the normalized_request instead of normalized_payload, so I think this will lead to an mismatch.

Reply:
Even if the case is not present since there is a possibility of occurring, I think this case should be handled. So you can split the case as default_keys in payload_arg_keys and default_keys in request_arg_keys, so that there is no mismatch.